bindings/node/test/clientEncryption.test.js

```diff
@@ -787,7 +790,8 @@ describe('ClientEncryption', function () {
     it('Case 4: can roundtrip encrypted indexed', async function () {
       const payload = await clientEncryption.encrypt('encrypted indexed value', {
         keyId: KEY1_ID,
-        algorithm: 'Indexed'
+        algorithm: 'Indexed',
+        contentionFactor: 10
```

Comment:
I don’t think it really matters since this test is never inserting data, but the spec says `0` here as well?